libraries/script-engine/src/ScriptCache.cpp

```diff
@@ -28,7 +27,7 @@ ScriptCache::ScriptCache(QObject* parent) {
 }
 
 QString ScriptCache::getScript(const QUrl& url, ScriptUser* scriptUser, bool& isPending, bool reload) {
-    assert(!_scriptCache.contains(url) || !reload);
+    //assert(!_scriptCache.contains(url) || !reload);
```

Comment:
This is a weird assert, it doesn't make sense to me. It fails if they are both true, but why would a caller be concerned with that?